content/en/docs/File Format/implementationstatus.md

```diff
@@ -13,94 +13,96 @@ implementations.
 The value in each box means:
 * ✅: supported
 * ❌: not supported
+* (R/W): partial reader/writer support
 * (blank) no data
 
 Implementations:
 * `C++`: [parquet-cpp](https://github.com/apache/arrow/tree/main/cpp/src/parquet)
 * `Java`: [parquet-java](https://github.com/apache/parquet-java)
 * `Go`: [parquet-go](https://github.com/apache/arrow-go/tree/main/parquet)
 * `Rust`: [parquet-rs](https://github.com/apache/arrow-rs/blob/main/parquet/README.md)
+* `CUDA`:[cudf](https://github.com/rapidsai/cudf)
```

Comment:
Should this say CUDA C++, or CUDA (C++/Python/Java), or something like that to indicate the language bindings from which cudf APIs can be called?
```suggestion
* `CUDA`: [cudf](https://github.com/rapidsai/cudf)
```

Reply:
From what I remember from https://github.com/apache/parquet-site/pull/34, this website is to indicate the language actually implementing the backend and other implementations may have similar bindings.

Reply:
Although, i am inclined to changing it to `CUDA C++` instead of just CUDA